src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -24,6 +24,7 @@
         "twig/twig": "~1.28|~2.0"
     },
     "require-dev": {
+        "symfony/console": "~3.3",
```

Comment:
the issue is that FrameworkBundle **always** registers the compiler pass coming from the component. So it means that the component **is required**. There is no point making the dependency optional in composer when it is only optional if you never use the bundle